### ISSUE TITLE
Death Certificate and Glitched Crown should both be 0.5 weight in the secret pool pool, maybe even lower.

### ISSUE THREAD

**Issue by crossstoney** [UnderscoreKilburn/repentance-issues#1716]
It's bizarre how Sacred Orb and Vanishing Twin are both 0.5 in the secret room pool while Death Certificate and Glitched Crown are both normal weight. Granted, the former two are not exclusive to the secret room pool, but Glitched Crown especially if encountered early-mid game (and obviously Death Certificate) will both guarantee run wins that it makes less sense for them to be seen more easily than Sacred Orb.

**Comment by Turtier** [comment]
I compiled item weights in the past and some are kinda weird in general. Most weight changes involve items (usually powerful ones) that have a "main" pool and then appearances outside that pool get a weight hit, and I think this makes sense (mum's knife is a great example, devil is it's main, and it's 1.0 weight there, but in item room its 0.2 and mums chest its 0.1). The rest of the weight changes... some have categories that make sense but some feel completely arbitrary.
- Plan C is 0.2 weight in most pools, but its a troll item and its in 6 pools, fair enough, this makes sense
- Plum flute, 0.5, probably because you can guarantee it from baby plum, makes sense
- Character starting items / pocket actives, like dark arts, abyss, book of virtues, 0.5 in main pool, this seems to be an attempt at making the items rarer so that runs as tjudas, tapollyon, bethany etc are more unique, and this makes sense I think. Altho a little weird that berserk got hit, might be a leftover from when berserk was tsamsons pocket active in development (pretty sure I heard that in an interview), or maybe it's just still considered similar enough.
- Big 12 charge actives, Mega Blast and Delirious, 0.2 and 0.4 respectively (mega blast is lower but you see more devil items at once so it makes sense). I think these items are meant to be rare but I don't see why it doesn't apply to mega mush.
- Has 2 "main" pools? Sacred orb, vanishing twin, heartbreak. I would say this is a power balance, but vanishing twin is quality 2, so thats not it? Why aren't more items, like red key, in this list? Feels very arbitrary to me, my category is probably wrong but it makes even less sense without the category so...
- Lower weight as a power balance? Magic mush (could be because it can be dropped by mushrooms too, but mini mush isn't hit, so power is clearly part of the equation), PJ's (technically 2 pools, but chest pool is glorified boss pool, none of the other boss items got hit), fate, card reading. Card reading is probs the rating I can make the least sense of, I think its a great item but why is it the ONLY shop exclusive item to get a weight hit? Its not like its the most powerful shop exclusive item either, at least fate actually is stronger than all the default weight items in the chest pool, that makes a little sense, but card reading? I don't get it. Was it originally also in the secret room, then taken out but the weighting wasn't updated?

So basically, there is some precedent for items getting decreased weight purely on power level, but it feels arbitrary what items get hit and which ones don't. It feels like mega mush should definitely have been reduced weight similar to mega blast, and originally when I noticed it wasn't, alongside death certificate and r key, I assumed secret room must have wanted to have 1 weight across the board, considering its the rarest pool to naturally get a pedestal from, so I think that would make some sense and would be cool... but sacred orb and vanishing twin are the ONLY exceptions and it seems very strange.

TLDR Local isaac player writes an essay about item weights and still doesn't understand the decisions behind item weights.
